doc/classes/Dictionary.xml

```diff
@@ -11,17 +11,26 @@
 		Creating a dictionary:
 		[codeblocks]
 		[gdscript]
-		var my_dir = {} # Creates an empty dictionary.
-		var points_dir = {"White": 50, "Yellow": 75, "Orange": 100}
-		var another_dir = {
-		    key1: value1,
-		    key2: value2,
-		    key3: value3,
+		var my_dict = {} # Creates an empty dictionary.
+		var dict_variable_key = "Another key name"
+		var dict_variable_value = "value2"
+		var points_dict = {"White": 50, "Yellow": 75, "Orange": 100}
+		var another_dict = {
+		    "Some key name": "value1",
+		    dict_variable_key: dict_variable_value,
+		}
+
+		# Alternative Lua-style syntax.
+		# Doesn't require quotes around keys, but only string constants can be used as key names.
+		# Additionally, key names must start with a letter or an underscore.
+		# Here, `some_key` is a string literal, not a variable!
+		another_dict = {
+		    some_key = "value2",
```

Comment:
can we use different example here 
    some_key = 42 

for example, or something that is not a string in a value. . We are talking a lot about strings then we use string as value that may confuse people.